src/client/types.ts

```diff
@@ -91,8 +93,26 @@ export type FlowControl = {
       parallelism: number;
       /**
        * number of requests to activate per second with the same flow control key
+       *
+       * @deprecated use rate instead
        */
       ratePerSecond?: number;
+      /**
+       * number of requests to activate within the period with the same flow control key.
+       *
+       * Default period is a second.
+       */
+      rate?: number;
+      /**
+       * The time interval during which the specified `rate` of requests can be activated
```

Comment:
When I first read this i did not understand it. It's hard to understand the logic from the description. I only did after looking at the example. Can we add, e.g. "10 requests per 1h"?